AppConnectSwift/AppDelegate.swift

```diff
@@ -17,7 +17,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         let dir = NSFileManager.defaultManager().URLsForDirectory(NSSearchPathDirectory.DocumentDirectory, inDomains: NSSearchPathDomainMask.UserDomainMask).last
         let key = "12345678901234567890123456789012".dataUsingEncoding(NSUTF8StringEncoding)
         // TODO: Substitute the apiToken value with proper token
-        MDBabbage.startWithEnvironment(MDClientEnvironment.Validation, apiToken: "Random String", publicDirectory: dir, privateDirectory: dir, encryptionKey: key)
+        MDBabbage.startWithEnvironment(MDClientEnvironment.Sandbox, apiToken: "3d4a7f259255a9b9c69bad748447ee64", publicDirectory: dir, privateDirectory: dir, encryptionKey: key)
```

Comment:
Wait, we shouldn't have an actual token in the repo. This is a public facing repo.
This is for the final user to put in token, and the TODO should stay as a reminder to them.